machiavelli-cards/src/main/java/pl/jellytech/machiavelli/cards/services/CardServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package pl.jellytech.machiavelli.cards.services;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import pl.jellytech.machiavelli.cards.entities.Card;
+import pl.jellytech.machiavelli.cards.repositories.CardRepository;
+
+import java.util.List;
+
+
+@Service
+@Slf4j
+public class CardServiceImpl implements CardService {
+    private final CardRepository cardRepository;
+
+    @Autowired
+    public CardServiceImpl(CardRepository cardRepository) {
+        this.cardRepository = cardRepository;
+    }
+
+    @Transactional
+    public Card createOrUpdate(Card card) {
+        card = this.cardRepository.save(card);
+        return card;
+    }
+
+    public Card getById(Long cardId) {
+        return this.cardRepository.getReferenceById(cardId);
+    }
+
+    public List<Card> getAll() {
```

Comment:
Can i will create it If that kind of query will be required on the game development?

Reply:
ok, we can add it in another PR
But I think it will be required before game dev even ;) 
Cause even for the card admin panel you'll want grouping per type